src/index.ts

```diff
@@ -96,8 +100,13 @@ export class WaxJS {
     ) => void;
     metricURL?: string;
     returnTempAccounts?: boolean;
+    chainName?: string;
```

Comment:
add chainId and registryEndpoint and default them to null

Reply:
updated 👍